tests/load/test_read_interfaces.py

```diff
@@ -51,6 +52,14 @@ def _expected_chunk_count(p: Pipeline) -> List[int]:
 @pytest.fixture(scope="session")
 def populated_pipeline(request) -> Any:
     """fixture that returns a pipeline object populated with the example data"""
+
+    # ensure ibis is installed for these tests
+    import subprocess
+
+    subprocess.check_call(
```

Comment:
this is hardcore. arent we able to use dependency group for ibis?

Reply:
I basically want to avoid running some subset of tests only with ibis, but I'll just add a group for ibis and make it available for all destination tests.

Reply:
done